pages/docs/features/inngest-functions/cancellation/cancel-on-timeouts.mdx

```diff
@@ -0,0 +1,129 @@
+import { CodeGroup, Row, Col, GuideSelector, GuideSection } from "src/shared/Docs/mdx";
+export const description = 'Learn how to cancel long running functions with events.'
+
+# Cancel on timeouts
+
+It's possible to force runs to cancel if they take too long to start, or if the runs execute for too long.  The `timeouts` configuration property allows you to automatically cancel functions based off of two timeout properties:
+
+- `timeouts.start`, which controls how long a function can stay unstarted in the queue
+- `timeouts.finish`, which controls how long a function can execute once started
+
+In the following examples, we'll explore how to configure the timeout property and how this works.
+
+
+<GuideSelector
+  options={[
+    { key: "typescript", title: "TypeScript" },
+    {/* { key: "python", title: "Python" }, */},
+    { key: "go", title: "Go" },
+  ]}>
+
+
+### `timeouts.start` - Adding timeouts to unstarted runs
```

Comment:
@tonyhb, suggestion: we might want to wrap also the content surrounding the code samples in `<GuideSection>` as Go configuration properties have a different name. wdyt?

Reply:
I thought about that, but didn't know if capitalization was such a big deal here.  I can, if we don't mind the repeated content.  Your call.